include/swift/AST/Decl.h

```diff
@@ -2282,8 +2282,8 @@ class ValueDecl : public Decl {
   /// \sa getFormalAccess
   /// \sa isAccessibleFrom
   AccessScope
-  getFormalAccessScope(const DeclContext *useDC = nullptr,
-                       bool respectVersionedAttr = false) const;
+  getFormalAccessScope(const DeclContext *useDC=nullptr,
+                       bool isUsageFromInline=false) const;
```

Comment:
(repeating comment here) This isn't really the right name either. It doesn't matter where this is being used from; it matters why the question's being asked.

Reply:
I can’t really think of anything better. Any suggestions?

Reply:
Maybe just go with a blunt `treatUsableFromInlineAsPublic`.